packages/expo-image-loader/ios/EXImageLoader/EXImageLoader.m

```diff
@@ -2,6 +2,7 @@
 
 #import <EXImageLoader/EXImageLoader.h>
 #import <React/RCTImageLoaderProtocol.h>
+#import <ExpoModulesCore/EXUtilities.h>
```

Comment:
was missing and the module was not registered - led to some import failures (same as above)

Reply:
cc @tsapeta